templates/template_db.go

```diff
@@ -24,8 +24,8 @@ func FindTemplateByName(ctx context.Context, key string, accountID string) (*Ins
 		return nil, false
 	}
 
-	sqlStatement := `SELECT id, template_name, package, image_id, account_id, firewall_enabled, networks, COALESCE(metadata,''), userdata, COALESCE(tags,'')
-FROM triton.tsg_templates
+	sqlStatement := `SELECT id, template_name, package, image_id, account_id, firewall_enabled, networks, COALESCE(metadata,''), userdata, COALESCE(tags,''), created_at
```

Comment:
Do we want to break on the back-tick (`) like the other statements? This is more for a sake of consistency.